src/raven.js

```diff
@@ -341,10 +341,11 @@ Raven.prototype = {
             return;
         }
 
+        msg = msg + ''; // Make sure it's actually a string
         // Fire away!
         this._send(
             objectMerge({
-                message: msg + ''  // Make sure it's actually a string
+                message: msg,
```

Comment:
Why did this change? And also introduced a trailing comma. I think that's still a bad thing?